src/geometry/euclidean/triangle.lean

```diff
@@ -352,4 +352,47 @@ begin
                                                 (λ he, h2 (vsub_eq_zero_iff_eq.1 he))
 end
 
+/-- Stewart's Theorem. -/
+theorem sq_dist_mul_dist_add_sq_dist_mul_dist (a : P) {b c p : P} (h : ∠ b p c = π) :
```

Comment:
```suggestion
theorem sq_dist_mul_dist_add_sq_dist_mul_dist (a b c p : P) (h : ∠ b p c = π) :
```
If you make all the points explicit arguments then it's still easy to apply the lemma, even if you don't have `h` ready.
Users might want `h` to become a new goal.